samples/helloworldasync/helloworld.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+	<LangVersion>latest</LangVersion>
```

Comment:
I think I would prefer explicitly specifying that this requires version 7.1, to make it clear which version introduced this option. (And in theory, if someone tried to compile this code with a 7.0 compiler, using `latest` would fail here.)

Also nit: tabs vs. spaces.

Reply:
Done.